frontend/src/views/Layout/DashboardLayout_gdd.spec.js

```diff
@@ -72,73 +72,23 @@ describe('DashboardLayoutGdd', () => {
       wrapper = Wrapper()
     })
 
+    it('has a navbar', () => {
+      expect(wrapper.find('.main-navbar').exists()).toBeTruthy()
+    })
+
     it('has a sidebar', () => {
-      expect(wrapper.find('nav#sidenav-main').exists()).toBeTruthy()
+      expect(wrapper.find('.main-sidebar').exists()).toBeTruthy()
     })
 
     it('has a main content div', () => {
       expect(wrapper.find('div.main-content').exists()).toBeTruthy()
     })
 
     it('has a footer inside the main content', () => {
-      expect(wrapper.find('div.main-content').find('footer.footer').exists()).toBeTruthy()
+      expect(wrapper.find('div.main-page').find('footer.footer').exists()).toBeTruthy()
     })
 
     describe('navigation bar', () => {
-      let navbar
-
-      beforeEach(() => {
-        navbar = wrapper.findAll('ul.navbar-nav').at(0)
-      })
-
-      it('has four items in the navbar', () => {
-        expect(navbar.findAll('ul > a')).toHaveLength(4)
-      })
-
-      it('has first item "overview" in navbar', () => {
-        expect(navbar.findAll('ul > a').at(0).text()).toEqual('overview')
-      })
-
-      it('has first item "overview" linked to overview in navbar', () => {
-        expect(navbar.findAll('ul > a > a').at(0).attributes('href')).toBe('/overview')
-      })
-
-      it('has second item "send" in navbar', () => {
-        expect(navbar.findAll('ul > a').at(1).text()).toEqual('send')
-      })
-
-      it('has second item "send" linked to /send in navbar', () => {
-        expect(wrapper.findAll('ul > a > a').at(1).attributes('href')).toBe('/send')
-      })
-
-      it('has third item "transactions" in navbar', () => {
-        expect(navbar.findAll('ul > a').at(2).text()).toEqual('transactions')
-      })
-
-      it('has third item "transactions" linked to transactions in navbar', async () => {
-        expect(wrapper.findAll('ul > a > a').at(2).attributes('href')).toBe('/transactions')
-      })
-
-      it('has fourth item "My profile" in navbar', () => {
-        expect(navbar.findAll('ul > a').at(3).text()).toEqual('site.navbar.my-profil')
-      })
-
```

Comment:
What happend to all of this? Why is the logout button still tested but the rest is misisng?

Reply:
dies wird in der Navbar.vue getestet